spinalcordtoolbox/deepseg/models.py

```diff
@@ -0,0 +1,94 @@
+# coding: utf-8
+"""
+Deals with models for deepseg module. Available models are listed under MODELS.
+"""
+
+
+import os
+import json
+import logging
+
+import spinalcordtoolbox as sct
+import spinalcordtoolbox.download
+
+
+logger = logging.getLogger(__name__)
+
+# List of models. The convention for model names is: (species)_(university)_(contrast)_region
+# Regions could be: sc, gm, lesion, tumor
+# TODO: add mirror
+# TODO: add a test to make sure all fields are present in the dict below
+# TODO: deal with "Folder __MACOSX"
+MODELS = {
+    't2star_sc':
+        {'url': 'https://osf.io/v9hs8/download?version=2',
+         'description': 'Cord segmentation on T2*-weighted contrast.',
+         'default': True},
```

Comment:
This dist has a `__MACOSX/` folder in it. The other two are okay though.

```
$ curl -JLO https://osf.io/v9hs8/download?version=2
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   459  100   459    0     0   1669      0 --:--:-- --:--:-- --:--:--  1675
100 17.1M  100 17.1M    0     0  3229k      0  0:00:05  0:00:05 --:--:-- 3821k
curl: Saved to filename 't2star_sc.zip'
$ unzip t2star_sc.zip 
Archive:  t2star_sc.zip
   creating: t2star_sc/
  inflating: t2star_sc/t2star_sc.pt  
   creating: __MACOSX/
   creating: __MACOSX/t2star_sc/
  inflating: __MACOSX/t2star_sc/._t2star_sc.pt  
  inflating: t2star_sc/t2star_sc.json  
  inflating: __MACOSX/t2star_sc/._t2star_sc.json  
  inflating: __MACOSX/._t2star_sc  
```

Reply:
will update the archive without this ugly addition once @charleygros has looked into [this problem](https://github.com/neuropoly/spinalcordtoolbox/pull/2639#issuecomment-629398909)

Reply:
fixed in e400d4dbf89611d8f80b9c0df70256222a11bc89